project/source/Scripts/Objects/CurrentSource.gd

```diff
@@ -81,9 +81,9 @@ func _on_RunCurrent_pressed():
 					print("At least one terminal is disconnected")
 					break
 			
-			#running = !running
-			#ToggleRunCurrentText()
-			#ToggleInputsEditable()
+			running = !running
```

Comment:
There's a bug with this branch where the Current Source's Stop button doesn't seem to work quite right. It stops it when clicked, but the text doesn't change and you (sometimes?) have to click it multiple times to get it to restart.

Keeping these lines commented out seems to prevent that problem, but this object isn't really my area of expertise so @tylerhand1, I'll leave this up to you, if you have time?